test/integration/framework/metacontroller.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2019 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	apiextensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+
+	"metacontroller.app/apis/metacontroller/v1alpha1"
+)
+
+// CreateCompositeController generates a test CompositeController and installs
+// it in the test API server.
+func (f *Fixture) CreateCompositeController(name, syncHookURL string, parentCRD, childCRD *apiextensions.CustomResourceDefinition) *v1alpha1.CompositeController {
```

Comment:
Is there any reason we don't return `(*v1alpha1.CompositeController, error)` here? In other words, why `Fatal` is preferred than `error`?

Reply:
I see. Thanks for the explanation!